src/mysql.php

```diff
@@ -53,11 +55,12 @@ function mysql_connect(
                 $password = ini_get('mysqli.default_pw') ?: null;
             }
 
-            $hash = sha1($hostname . $username . $flags);
+            $hash = sha1($hostname.$username.$flags);
             /* persistent connections start with p: */
             if ($hostname{1} !== ':' && isset(\Dshafik\MySQL::$connections[$hash])) {
                 \Dshafik\MySQL::$last_connection = \Dshafik\MySQL::$connections[$hash]['conn'];
                 \Dshafik\MySQL::$connections[$hash]['refcount'] += 1;
+
```

Comment:
Whitespace before return is good, I sometimes omit it if it's a very short block, but consistency is good.